caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
It is. Was going to dev it on another branch, so branches for: 
1. list view - thumbnail view ajax
2. email search and manage listings page as a render of the index page.